<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test fork PblSqlRelationalTableModel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainLo">
     <property name="margin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="cmb_Language">
         <item>
          <property name="text">
           <string>en</string>
          </property>
          <property name="icon">
           <iconset resource="../pbltableview/resource2.qrc">
            <normaloff>:/icon/img/countries/english-100x100.png</normaloff>:/icon/img/countries/english-100x100.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ru</string>
          </property>
          <property name="icon">
           <iconset resource="../pbltableview/resource2.qrc">
            <normaloff>:/icon/img/countries/russian-100x100.png</normaloff>:/icon/img/countries/russian-100x100.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>zh</string>
          </property>
          <property name="icon">
           <iconset resource="../pbltableview/resource2.qrc">
            <normaloff>:/icon/img/countries/china-100x100.png</normaloff>:/icon/img/countries/china-100x100.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="btn_goods">
         <property name="text">
          <string>goods</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_checks">
         <property name="text">
          <string>checks</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_logView">
         <property name="text">
          <string>view log</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>document</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
        <item>
         <layout class="QVBoxLayout" name="tableViewLO">
          <property name="margin">
           <number>10</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="margin">
             <number>5</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_sum">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(85, 170, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(165, 183, 238);</string>
              </property>
              <property name="text">
               <string>Sum</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btn_save">
            <property name="text">
             <string>SAVE DOCUMENT</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="margin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_Close">
         <property name="text">
          <string>Close</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../pbltableview/resource2.qrc"/>
  <include location="../pbltableview/resource2.qrc"/>
 </resources>
 <connections/>
</ui>
